🐛config.md - Email Config bug fixed (#156)
kubesphere/notification-manager: docs/crds/config.md

- `wechatApiSecret` - The secret of the application which to send messages. For more information, please refer to [secret](https://developer.work.weixin.qq.com/document/path/90665#secret).
- `wechatApiAgentId` - The id of the application which to send messages. For more information, please refer to [agentid](https://developer.work.weixin.qq.com/document/path/90665#agentid).

> > Any user, party or tag who wants to receive notifications must be in the allowed users list of the application which to send message.

Reviewer: Any user, party or tag who wants to receive notifications must be in the allowed users list of the application which to send message.
=>
> Any user, party, or tag that needs to be notified must be in the allowed user list of the application that sends the notification.

> > Any user, party or tag who wants to receive notifications must be in the allowed users list of the application which to send message.

Reviewer: ```suggestion
> Any user, party, or tag that needs to be notified must be in the allowed user list of the application that sends the notification.
```
